lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+    context['certificate_info_description'] = _("{company_name} acknowledges achievements through certificates, which "
+                                                "are awarded for various activities {company_name} students complete "
+                                                "under the {honor_code_html}.  Some certificates require completing "
+                                                "additional steps, such as {verifying_html}.").format(
+        company_name=company_name,
+        honor_code_html=cert_info_honor_code_html,
+        verifying_html=cert_info_verified_html
+    )
+
+    # Translators:
+    context['certificate_type_title'] = _('{certificate_type} Certfificate').format(
+        certificate_type=context.get('certificate_type')
+    )
+
+    # Translators:
+    context['certificate_verify_title'] = _("How {company_name} Validates Student Certificates").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['certificate_verify_description'] = _("Certificates issued by {company_name} are signed by a gpg key so "
+                                                  "that they can be validated independently by anyone with the "
+                                                  "{company_name} public key. For independent verification, "
+                                                  "{company_name} uses what is called a "
+                                                  "&quot;detached signature&quot;.").format(company_name=company_name)
+
+    # Translators:
+    context['certificate_verify_urltext'] = _("Validate this certificate for yourself")
+
+    # Translators:
+    context['company_about_description'] = _("{company_name} offers interactive online classes and MOOCs from the "
+                                             "world's best universities, including MIT, Harvard, Berkeley, University "
+                                             "of Texas, and many others.  {company_name} is a non-profit online "
+                                             "initiative created by founding partners Harvard and MIT.").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['company_about_title'] = _("About {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_about_urltext'] = _("Learn more about {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_courselist_urltext'] = _("Learn with {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_careers_urltext'] = _("Work at {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_contact_urltext'] = _("Contact {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_privacy_urltext'] = _("Privacy Policy")
+
+    # Translators:
+    context['company_tos_urltext'] = _("Terms of Service &amp; Honor Code")
```

Comment:
Would be great if we can escape a literal ampersand after we return from the translation function